node/core/approval-voting/src/lib.rs

```diff
@@ -1867,6 +1867,7 @@ fn process_wakeup(
 // Launch approval work, returning an `AbortHandle` which corresponds to the background task
 // spawned. When the background work is no longer needed, the `AbortHandle` should be dropped
 // to cancel the background work and any requests it has spawned.
+// TODO How do we leverage this for prevalidation of collators
```

Comment:
This is the specific point where the PVF is fetch on-chain. How would we go about exposing this to enable fetching the PVF for pre-validation?

It seems like the best option is to duplicate the code below somewhat to ensure that the pvf code is accessible to the CandidateValidation subsystem for prevalidation.